L1Trigger/TrackFindingTMTT/src/Settings.cc

```diff
@@ -45,6 +45,7 @@ namespace tmtt {
         kalmanRemove2PScut_(true),
         kalmanMaxSkipLayersHard_(1),  // On "hard" input tracks
         kalmanMaxSkipLayersEasy_(2),  // On "easy" input tracks
+        KFUseMaybeLayers_(true),
```

Comment:
Lines 47 & 48 need swapping over. Compiler complains that initialization takes place in different order to data member declaration in header file.